### DISCUSSION TITLE
How do I get the path to the current log file?

### DISCUSSION THREAD

**Discussion by jjeff** [megahertz/electron-log#266] [Q&A] [answered]
I'm trying to add a menu item to show the log file in the OS. Something like this:
```javascript
shell.showItemInFolder(app.getPath('logs'))
```
...but with the electron-log specific path to the (current?) log file.

How would I get that value?

I'm guessing it's going to be something like
```javascript
shell.showItemInFolder(log.transports.file.filePath)
```
but I can't seem to figure it out.

**Comment by megahertz** [comment, marked as answer]
By default, there are many log files for different process, so I would prefer `shell.openPath(path.dirname(log.transports.file.getFile().path))`

**Reply by jjeff** [reply]
Works perfectly! Thanks!
